api/apps/articles/routes.coffee

```diff
@@ -16,6 +16,7 @@ User = require '../users/model.coffee'
       'Pass published=true to only view published articles.')
 
   Article.where req.query, (err, results) ->
+    debug err if err
```

Comment:
I would think [this `debug` in the error handler middleware](https://github.com/artsy/positron/blob/master/api/lib/middleware.coffee#L25) would take care of that no?